src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/CABI.java

```diff
@@ -50,6 +60,16 @@
             }
         } else if (arch.equals("aarch64")) {
             current = AArch64;
+        } else if (arch.startsWith("ppc64")) {
+            if (os.startsWith("Linux")) {
+                current = SysVppc64le;
+            } else {
+                current = AIX;
+            }
```

Comment:
We don't support ppc64 Linux BE (given there are only ppc64le Linux and AIX in open sourced builds on Power) in which case there is no need to check the byte order for Linux; otherwise we need to add new code to support if required.